src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -553,6 +554,20 @@ static Library()
                     returnBehavior: ReturnBehavior.ReturnBlankIfAnyArgIsBlank,
                     targetFunction: ForAll)
             },
+            {
+                BuiltinFunctionsCore.ForAll_UO,
+                StandardErrorHandlingAsync<FormulaValue>(
+                    expandArguments: NoArgExpansion,
+                    replaceBlankValues: DoNotReplaceBlank,
+                    checkRuntimeTypes: ExactSequence(
+                        ExactValueTypeOrBlank<UntypedObjectValue>,
+                        ExactValueTypeOrBlank<LambdaFormulaValue>),
```

Comment:
>ExactValueTypeOrBlank<LambdaFormulaValue> [](http://example.com/codeflow?start=24&length=41)

Can we ever have a null lambda formula value? Even in an expression like `ForAll([1,2,3], Blank())` the second parameter isn't null, it's a lambda that returns a null value

Reply:
Unfortunately, all the other functions in this file use that runtime type check. I can fix it in a future PR.